dev/cognitiveservices/data-plane/Language/analyzeconversations.json

```diff
@@ -2096,6 +2105,201 @@
         "text"
       ]
     },
+    "AnalyzeConversationalSentimentTask": {
+      "type": "object",
+      "description": "Task definition for a sentiment analysis in conversations.",
+      "properties": {
+        "parameters": {
+          "$ref": "#/definitions/ConversationalSentimentTaskParameters"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationLROTask"
+        }
+      ],
+      "x-ms-discriminator-value": "ConversationalSentimentTask"
+    },
+    "ConversationalSentimentTaskParameters": {
+      "type": "object",
+      "description": "Supported parameters for a Conversational sentiment analysis task.",
+      "properties": {
+        "predictionSource": {
+          "type": "string",
+          "description": "For transcript conversations, this parameter provides information regarding which content type (ITN, Text, Lexical, Masked ITN) should be used for sentiment analysis. The details of the sentiment analysis - like the offset, length and the text itself - will correspond to the text type selected here.",
```

Comment:
If `TranscriptContentType` is an enum (which I hope it is but haven't gotten there yet) I don't recommend documenting all its values. The enum alone does that and then you only have one place to keep up to date.

Reply:
Good point!